science/kicad/Portfile

```diff
@@ -121,6 +124,7 @@ if {${name} eq ${subport}} {
 }
 
 subport kicad-docs {
+    version             8.0.0-rc3
```

Comment:
This is taken from a third-party source. Alternative would be to generate documentation from source.